src/Ryujinx.Audio/Renderer/Server/Splitter/SplitterDestinationVersion2.cs

```diff
@@ -110,7 +111,8 @@ public void Update<T>(in T parameter) where T : ISplitterDestinationInParameter
 
                 _biquadFilters = parameter.BiquadFilters;
 
-                if (!IsUsed && parameter.IsUsed)
+                bool resetPrevVoume = isPrevVolumeResetSupported ? parameter.ResetPrevVolume : !IsUsed && parameter.IsUsed;
```

Comment:
```suggestion
                bool resetPrevVolume = isPrevVolumeResetSupported ? parameter.ResetPrevVolume : !IsUsed && parameter.IsUsed;
```
Need to change the use below, too.